### PR TITLE
Delta token changes every time you ask for change-set

### PR DESCRIPTION
**Pull request by geekwhocodes**
Delta token changes every time you ask for change-set using previous delta token. I have tested it with `v1.0` API version.

### CODE REVIEW

**Comment by Lauragra** [comment]
@piotrci, please review this PR. If approved, we'll create a new PR as this one is conflicting.

File: concepts/delta_query_users.md [hunk 1]
```diff
@@ -145,28 +145,28 @@ GET https://graph.microsoft.com/v1.0/users/delta?$deltatoken=oEcOySpF_hWYmTIUZBO
 
 ## deltaLink response
 
-If no changes have occurred, the same `deltatoken` is returned with no results.
+If no changes have occurred, the differet `deltatoken` is returned with no results.
```
Comment (baywet) [inline concepts/delta_query_users.md]:
```suggestion
If no changes have occurred, a different `deltatoken` is returned with no results.
```

File: concepts/delta_query_users.md [hunk 2]
```diff
@@ -145,28 +145,28 @@ GET https://graph.microsoft.com/v1.0/users/delta?$deltatoken=oEcOySpF_hWYmTIUZBO
...
 {
   "@odata.context":"https://graph.microsoft.com/v1.0/$metadata#users",
-  "@odata.deltaLink":"https://graph.microsoft.com/v1.0/users/delta?$deltatoken=oEcOySpF_hWYmTIUZBOIfPzcwisr_rPe8o9M54L45qEXQGmvQC6T2dbL-9O7nSU-njKhFiGlAZqewNAThmCVnNxqPu5gOBegrm1CaVZ-ZtFZ2tPOAO98OD9y0ao460",
+  "@odata.deltaLink":"https://graph.microsoft.com/v1.0/users/delta?$deltatoken=oEcOySpF_hWYmTIUZBOIfPzcwisr_rPe8o9M54L45qEXQGmvQC6T2dbL-9O7nSU-njKhFiGlAZqewNAThmCVnNxqPu5gOBegrm1CaVZ-ZtFZ2tPOAO98OD9y0ao461",
   "value": []
 }
 ```
 
-If changes have occurred, the same `deltatoken` is returned including a collection of changed users.
+If changes have occurred, the differet `deltatoken` is returned including a collection of changed users.
```
Comment (baywet) [inline concepts/delta_query_users.md]:
```suggestion
If changes have occurred, a different `deltatoken` is returned including a collection of changed users.
```

**Comment by baywet** [comment]
@Lauragra thanks for pointing me to this PR, I tested the suggested changes and they are correct. I also fixed a few typos.
For some reason I don't have the permission to resolve conflicts on this PR, do you have permissions to do so?
@geekwhocodes thanks for your contributions! please disregard my previous comment asking you to integrate changes, once my review was submitted I was able to add the changes myself.

**Comment by Lauragra** [comment]
I recreated this update in PR #7095; this PR is out of date and that's why we can't resolve the conflict. Closing this one and I'll merge in the other. Thanks.

**Pull request closed without merging**